lib/cache.js

```diff
@@ -79,5 +78,18 @@ var save = cache.save = function (cb) {
  * @return {Promise}
  */
 cache.clear = function (cb) {
-  return unlinkAsync(cache.source).nodeify(cb);
+  fs.unlink(cache.source, function (err) {
+    // if we weren't given a callback, it doesn't matter
+    if (!cb) {
+      return;
+    }
+
+    // swalling the error if it's a file does not exist error
+    if (err.code === 'ENOENT') {
```

Comment:
We should check, that `err` is actually defined. In case of success this would get a `cannot call code on undefined` exception.